docs/spec/spec.md

```diff
@@ -447,6 +447,44 @@ The `rabbitmq:Message` record received can be used to retrieve its contents.
       BasicProperties properties?;
    |};
 ```
+Subtypes of `rabbitmq:AnydataMessage` can be used to bind data to a specific type.
+```ballerina
+    public type StringMessage record {|
+       *rabbitmq:AnydataMessage;
+       string content;
+   |};
+   rabbitmq:Service listenerService =
+   @rabbitmq:ServiceConfig {
+      queueName: "MyQueue"
+   }
+   service object {
+      remote function onRequest(StringMessage message) returns string {
+         return "Hello Back!";
```

Comment:
ditto

Reply:
fixed in https://github.com/ballerina-platform/module-ballerinax-rabbitmq/pull/399/commits/3a0c50ad81de8fcbe2dab219b7971de22ca3be2e